packages/acgtk/acgtk.1.5.3/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+build: [
+  ["dune" "subst"] {dev}
+# remove the -p to also build the local libraries: conflict with the
+# fact that some libraries are also part of the acgtkLib package
```

Comment:
Does this require to add a `conflicts: ["acgtklib"]` section?

Reply:
I don't think so. The acgtkLib package was only meant for local dev usage and is not meant to be distributed with opam anyway, but I never dig deeply enough in opam+dune to clearly understand what the problem was. Anyway, it is not used anymore.